src/main/java/billennium/tests/entity/Answer.java

```diff
@@ -24,7 +24,8 @@
     @JsonIgnore
     private Question question;
 
-    @org.hibernate.annotations.Type(type="true_false")
+    @org.hibernate.annotations.Type(type="true_false") // TODO dlaczego adnotacja w taki sposob ? lepszy import niz
+    // qualified wydaje mi sie. Dlaczego correctAnswer jest primitywem ?
```

Comment:
nie widziałem przewiskazań by urzyć prymitywa. Zajmuje trochę mnie pamięci i w sumie chciałem sprawdzić czy zadziałą to w ten sposób. To bardziej dla próby.